### ISSUE TITLE
Question: viability of android browser as client?

### ISSUE THREAD

**Issue by thiswillbeyourgithub** [jlesage/docker-baseimage-gui#149]
Hi,

If I understand correctly, this project can be used to easily turn an application into an interactive website. Meaning I can use any browser, connect to the appropriate URL and access the app.

My question was, can this be used to access reliably a desktop app from an android phone? Like, if we get the screen resolution and verticality out of the way, does the keyboard gets raised if I select a field in the browser? Does sound get transfered to the phone too?

If the answer is yes, I think it can be of tremendous value by quite easily making android friendly app out of desktop apps!

Thanks :)

**Comment by jlesage** [comment]
Yes, the web interface allows you to open the keyboard when needed (but it doesn't open automatically when selecting a text field inside the Linux application) and sound does work. However, keep in mind that you are still accessing a Linux application designed for desktop environment. Thus, the experience on a phone can be impacted.

**Comment by thiswillbeyourgithub** [comment]
Thanks! Also I ended up wondering, given that there are on screen keyboard that exist on linux, doesn't that mean there's a way to know when a field was triggered? Or maybe the on screen keyboard has to be opened manually? I don't know.

**Comment by jlesage** [comment]
I tried to find more information about that, but I think that in most case the onscreen keyboard is opened manually. I found something about "gnome accessibility" that might trigger the onscreen keyboard automatically when clicking in a text field... but it doesn't seem to have a "standard" way in linux to detect a such action.

**Comment by thiswillbeyourgithub** [comment]
That's too bad. Well thank you very much anyway :)

**Comment by jlesage** [comment]
But in any case, currently the virtual keyboard can be opened manually.

**Comment by thiswillbeyourgithub** [comment]
Hello again. I thought some more about it.

What about if we had a floating button just like on the desktop that trigger the virtual keyboard but here it would instead send the browser command to trigger the android keyboard? Btw i'm using [unexpected keyboard](https://github.com/Julow/Unexpected-Keyboard) which includes modifier keys like ctrl, fn, meta etc.

> But in any case, currently the virtual keyboard can be opened manually.

Oh wait do you mean trigger the desktop "virtual keyboard" or the android virtual keyboard? The latter is of course much more adapted on android I would say. No?
